javascript/ql/src/Comments/CommentedOutCode.qhelp

```diff
@@ -2,6 +2,6 @@
   "-//Semmle//qhelp//EN"
   "qhelp.dtd">
 <qhelp>
-  <include src="CommentedOutCodeQuery.qhelp" />
+  <include src="CommentedOutCodeQuery.inc.qhelp" />
   <include src="CommentedOutCodeCommon.qhelp" />
```

Comment:
This one also needs a rename in this PR, since it's being changed.

Reply:
It needs a rename because 1) this file is changed which references it and 2) there is no corresponding `.ql` file, right?
I actually had already updated this by accident (but forgot the file rename), then reverted it and moved it to #5275 🤦

Is there a built script I can run to notice these errors myself or will it only work through trial and error by you having to check the internal CI errors?

Reply:
Have amended this change to the last commit and force-pushed. There are no other changes so the [diff](https://github.com/github/codeql/compare/45137852038fcfa3630e39d47e61ac594d7c1613..6f70ac570c8ec78919aa19050d99aa6d6cfeec7d) only shows these changes.

Reply:
> It needs a rename because 1) that file is changed and 2) there is no corresponding .ql file, right?

Correct.

> Is there a built script I can run to notice these errors myself or will it only work through trial and error by you having to check the internal CI errors?

Unfortunately not. The bulk qhelp processing actually doesn't require these renames at the moment - it is only the CI that chokes on the the `.qhelp`-without-`.ql` situation.

Reply:
Looks like there's a similar situation for `javascript/ql/src/Metrics/FLinesOfSimilarCodeCommon.qhelp`

Reply:
Sorry that I missed that. I have force-pushed the changes and went through all changed files to make sure that I have not missed another case, but I did not find any.